dashboard/app/graphs.go

```diff
@@ -176,20 +203,23 @@ func loadGraphBugs(c context.Context, ns string) ([]*Bug, error) {
 				continue
 			}
 			bugReporting := lastReportedReporting(bug)
-			if bugReporting == nil || bugReporting.Auto && bug.Status == BugStatusInvalid {
+			if removeDupInvalid &&
```

Comment:
What was the intent of the `bugReporting == nil` --> `removeDupInvalid && bugReporting == nil` change?

Reply:
I missed parenthesis, this was intended to be:
```
if removeDupInvalid &&
				(bugReporting == nil || bugReporting.Auto && bug.Status == BugStatusInvalid) {
```

I excluded auto-obsoleted bugs b/c they are obsoleted with significant delay, so last 1/2 months will always look much better than the previous ones. So we won't be able to track recent efficiency.

Fixed now.